### DISCUSSION TITLE
Clarius IMU rearranging image in 3D view

### DISCUSSION THREAD

**Discussion by stka85** [PlusToolkit/PlusLib#1145] [General]
Hello, I have connected the Clarius C3 HD3 to Slicer via the Plus Toolkit. According to the video https://www.youtube.com/watch?v=2vXeJxYFou4&t=4s, I expected the live image in the 3D view to rearrange itself while moving the scanner. I have also activated the IMU data transfer. Is there another setting that I am not aware of? Should this work?
Is there any documentation on how to adjust the config file?
Thank you!
[PlusDeviceSet_Server_ClariusVideoCapture_IMU_Tracking.txt](https://github.com/PlusToolkit/PlusLib/files/13706425/PlusDeviceSet_Server_ClariusVideoCapture_IMU_Tracking.txt)

**Comment by ungi** [comment]
In the video, an optical tracker was used for 6 degrees of freedom tracking of the ultrasound probe. I'm not sure if the Clarius IMU is supported in PLUS, but even if it is, that will not give you translational motion tracking. To reconstruct larger 3D ultrasound volumes, you will need an external tracker device besides the ultrasound.

**Comment by stka85** [comment]
Hello,
thank you so much for your response!
Slicer can receive the IMU Data. What is about the magnetic field and the 
tilt sensor data that is supported in Slicer. How can I connect the 
VideoStream with the TrackingStream ?
Thank you!


Am Mo, 18 Dez 2023 14:06:55 -0800
  schrieb Tamas Ungi ***@***.***>:

**Reply by ungi** [reply]
In the video, this tracker was used: https://www.optitrack.com/cameras/v120-duo/
If you use that same tracker, you can take the OptiTrack example config files in PLUS to make sure PLUS connects to your device. Add that device to your already working ultrasound configuration in the DataCollection section. Then, add a VirtualMixer device to the configuration to merge the tracker and video streams in a single tracked video stream. An example for mixer: https://github.com/PlusToolkit/PlusLibData/blob/master/ConfigFiles/PlusDeviceSet_Server_Ultrasonix_C5-2_Ascension3DG_calibrated.xml
